core/chains/evm/client/node_fsm.go

```diff
@@ -78,6 +84,8 @@ const (
 	// disconnected, then put into a revive loop and re-awakened after redial
 	// if a new head arrives
 	NodeStateOutOfSync
+	// NodeStateUnusable is a sendonly node that has an invalid URL that can never be reached
+	NodeStateUnusable
```

Comment:
Can you update the `README.md` mermaid diagram?

Reply:
Do you think it worth updating the primary node lifecycle with an "Unusable" state as well?